### DISCUSSION TITLE
Profiler service missing DefaultSection class

### DISCUSSION THREAD

**Discussion by litzinger** [ExpressionEngine/ExpressionEngine#2956] [Feature Requests]
**Description of the problem**
The Profiler service attempts to load a `Section\DefaultSection` class if trying to add a section that is not default/native EE.

`$class = __NAMESPACE__."\\Section\\DefaultSection";`

**How To Reproduce**
Add a non-default call to the addSection method, e.g.

```
ee('Profiler')->addSection('publisher', ['foo' => 'bar']);
```

**Environment Details:**
 - Version: 5.3 & 6.0a

**Comment by intoeetive** [comment]
@litzinger thanks for the report!

It does not look like Profiler service is currently available for the developer at all, so there might be some larger amount of work on this front. As such, classifying this as enhancement
